tests/integration_tests/test_observation_times.py

```diff
@@ -113,12 +113,14 @@ def test_small_time_mismatches_are_ignored(
         stderr = StringIO()
         with redirect_stderr(stderr):
             run_cli(
-                ES_MDA_MODE,
                 "--disable-monitor",
+                ES_MDA_MODE,
                 str(tmp_path / "config.ert"),
                 "--weights=0,1",
             )
-        assert "Experiment completed" in stderr.getvalue()
+        assert (
+            not stderr.getvalue()
+        ), "There should be no errors when mismatches are small"
```

Comment:
I don't think this will work, as the monitor will always print to the same stream (no matter failure or success):

https://github.com/equinor/ert/blob/6f49e28aa529c7d18de9513954ee848fa04d7b1b/src/ert/cli/monitor.py#L106-L115

So this will make the test pass, but I'm not sure it will fail if there is an actual error.